web-app/src/pages/Profile/index.jsx

```diff
@@ -30,15 +32,15 @@ const Profile = props => {
           <div className="columns">
             <div>
               <h1>{totalHolidays - daysBooked}</h1>
-              <h4>Days Remaining</h4>
+              <h4><FontAwesomeIcon icon={faCalendar} /> Days Remaining</h4>
```

Comment:
So in my eyes here it should be `<FontAwesomeIcon icon={faCalendar} /><h4>Days Remaining</h4>`

Easier to read and the header tag should only contain the header text. Same will apply for the others below.

Reply:
Just looking through the other files, it seems this has been done across the app. It would be cleaner to have it `<FontAwesomeIcon icon={faCalendar} /><h4>Days Remaining</h4>` but because it's been done already just leave this comment out.

Reply:
Its easier to have the font inside the h4 tag so that the icons inherits their styles. Possibly be tidier if the text was wrapped in a span tag? So it would be like this:

```
<h4>
  <FontAwesomeIcon icon={faCalendar} />
  <span>Days Remaining</span>
</h4>
```
Although if it's not done like that anywhere else, I agree with Gary, keep as is.

Reply:
Issue being it's not valid HTML but it still works of course. I wouldn't put the span in either. It ideally shouldn't have nested tags.